iib/workers/tasks/build.py

```diff
@@ -781,6 +803,19 @@ def _prepare_request_for_build(
         from_index_info['resolved_distribution_scope'], distribution_scope
     )
 
+    if not binary_image:
+        binary_image_ocp_version = from_index_info['ocp_version']
+        binary_image_distribution_scope = distribution_scope
```

Comment:
What is the purpose of this line?

Reply:
You're right. I assumed a different implementation when `distribution_scope` was included for merge-index-image. But looking at Jindrich's PR, this is not required.